tests/pre_merge/datasets/test_dataset.py

```diff
@@ -55,10 +55,8 @@ def test_random_split(self, folder_dataset):
         # check if reconstructed dataset is equal to original dataset
         assert folder_dataset.samples.equals(reconstructed_dataset.samples)
 
-        # check if warning raised when one of the subsets is empty
-        split_ratios = [1 - (1 / (len(folder_dataset) + 1)), 1 / (len(folder_dataset) + 1)]
-        with pytest.warns():
-            subsets = random_split(folder_dataset, split_ratios)
+        # TODO(djdameln): Check if warning raised when one of the subsets is empty
+        # CVS-122709
```

Comment:
Should we leave the removed code in comments? Otherwise we might forget what this todo was referring to

Reply:
hmm, one of the rules complain when there is a commented code, but I guess we could temporarily ignore it. Alternatively include that in the ticket

Reply:
Let's change this to
```suggestion
        # TODO(djdameln): Add check for handling of empty subsets in case of small split ratio CVS-122709
```

Reply:
I think this is the format suggested by the rule. Your suggestion would not work unfortunately. 
```
# TODO(djdameln): Check if warning raised when one of the subsets is empty
# CVS-122709
```